piqe_ocp_lib/tests/tasks/test_populate_ocp.py

```diff
@@ -106,6 +106,14 @@ def test_delete_operator_from_cluster_failed_to_get_sub(self, mock_get_sub, get_
 
         assert "Failed to retrieve subscription" in [m.message for m in caplog.records]
 
+
+    def test_verify_operator_installed(self, get_kubeconfig):
+        verify=OperatorInstaller(get_kubeconfig)
+        csv_obj = verify.verify_operator_installed('local-storage-operator','openshift-local-storage')
```

Comment:
Does this test assume the local-storage-operator is already installed? how is it present in the first place? I think this test should either install and delete some operator, otherwise you can check on a csv that is always present by default like 'packageserver' in the 'openshift-operator-lifecycle-manager'

Reply:
Done.